lib/Controller/PageController.php

```diff
@@ -141,51 +142,41 @@ public function showCall(string $token): Response {
 	}
 
 	/**
-	 * @PublicPage
-	 * @NoCSRFRequired
-	 *
 	 * @param string $token
 	 * @param string $password
 	 * @return Response
 	 * @throws HintException
 	 */
+	#[NoCSRFRequired]
+	#[PublicPage]
 	#[UseSession]
 	#[BruteForceProtection(action: 'talkRoomPassword')]
 	public function authenticatePassword(string $token, string $password = ''): Response {
 		// This is the entry point from the `/call/{token}` URL which is hardcoded in the server.
 		return $this->index($token, '', $password);
 	}
 
-	/**
-	 * @PublicPage
-	 * @NoCSRFRequired
-	 *
-	 * @return Response
-	 */
+	#[NoCSRFRequired]
+	#[PublicPage]
 	public function notFound(): Response {
 		return $this->index();
 	}
 
-	/**
-	 * @PublicPage
-	 * @NoCSRFRequired
-	 *
-	 * @return Response
-	 */
+	#[NoCSRFRequired]
+	#[PublicPage]
 	public function duplicateSession(): Response {
 		return $this->index();
 	}
 
 	/**
-	 * @PublicPage
-	 * @NoCSRFRequired
-	 *
 	 * @param string $token
 	 * @param string $callUser
 	 * @param string $password
 	 * @return TemplateResponse|RedirectResponse
 	 * @throws HintException
 	 */
+	#[NoCSRFRequired]
+	#[PublicPage]
```

Comment:
Maybe would be good put all ordered.

Reply:
We also can verify if possible create a rule at friendsofphp/php-cs-fixer about this.

Reply:
it's alphabetically ordered?

> We also can verify if possible create a rule at nextcloud/coding-standard about this.

I guess that would be cool